src/utils/is-test-mode.ts

```diff
@@ -7,3 +7,7 @@
 export const isTestMode = (): boolean => {
   return !!process.env.REACT_APP_TEST_MODE
 }
+
+export const isMockMode = (): boolean => {
```

Comment:
The name of the file is a bit misleading as it now doesn't contain the isTestMode exclusively.

Reply:
renamed to `test-modes`